docker/standalone.php

```diff
@@ -394,26 +395,15 @@ function initUI() {
 			flex-direction: column;
 			gap: 20px;
 			align-items: center; 
-			padding-top: 43px;
-		}
-
-		div.testArea {
-			display: flex;
-			width: 248px;
-			padding: 24px 33px 24px 24px;
-			flex-direction: column;
-			align-items: flex-start;
-			gap: 12px;
-
-			border-radius: 16px;
-			border: 1px solid var(--Monochrome-Grey-7, #D1D5DB);
-			background: var(--Monochrome-White, #FFF);
+			padding: 43px 20px 0 20px;
+			width: 100%;
 
 		}
 
-		div.testArea2 {
+		div.testArea, div.testArea2 {
 			display: flex;
-			width: 248px;
+			width:100%;
+			width: 250px;
```

Comment:
you have two width. Fix this.

Reply:
max-width: 250px; is required I am fixing this.